### PR TITLE
BD-3213 Add rules-based merging to Duplicate Users

### PR DESCRIPTION
**Pull request by rachel-feinberg**
# Pull Request/Issue Resolution

Closes #**BD-3213**

#### Is this change associated with a Braze feature/product release?
- [x] Yes (**MERGE JULY 7, 2024**)
- [ ] No

### CODE REVIEW

File: _docs/_user_guide/engagement_tools/segments/user_profiles/duplicate_users.md [hunk 1]
```diff
@@ -87,3 +87,33 @@ Duplicate user profiles cannot be recovered after merging.
...
+
+![The "Edit rules" panel with sections to select options for "Profile to keep" and "Resolving ties".]({% image_buster /assets/img/audience_management/duplicate_users/edit_rules.png %})
+
+For example, you could keep the profile that has a phone number. If multiple users have the same phone number, you could resolve ties using the **Updated date** field and prioritze the newest user.
+
+### Step 2: Preview the results (optional)
+
+After saving your rules, you can preview how they'll work by selecting **Generate a list of duplicates**. Braze will generate your preview and send it to your email address as a CSV file that shows which users would be kept and merged if your rules were applied. 
+
+### Step 3: Merge duplices
```
Comment (rachel-feinberg) [inline _docs/_user_guide/engagement_tools/segments/user_profiles/duplicate_users.md:113]:
```suggestion
### Step 3: Merge duplicates
```

File: _docs/_user_guide/engagement_tools/segments/user_profiles/duplicate_users.md [hunk 2]
```diff
@@ -87,3 +87,33 @@ Duplicate user profiles cannot be recovered after merging.
...
+
+### Step 1: Define your rules
+
+1. Go to **Audience** > **Manage Audience** > **Edit rules**.
+
+2. In the **Profile to keep** section of the **Edit rules** panel, select the **Identifier** for the profiles that will be kept when merging duplicates. This can be the email address or phone number.
+
+3. In the **Resolving ties** section, select the criteria that will determine how to solve ties between profiles with matching criteria from **Profile to keep**. You can select the following:
+- **Resolve ties using**: Created date, Updated date, Last session
+- **Prioritzation**: Newest, Oldest
```
Comment (rachel-feinberg) [inline _docs/_user_guide/engagement_tools/segments/user_profiles/duplicate_users.md:103]:
```suggestion
- **Prioritization**: Newest, Oldest
```

File: _docs/_user_guide/engagement_tools/segments/user_profiles/duplicate_users.md [hunk 3]
```diff
@@ -87,3 +87,31 @@ Duplicate user profiles cannot be recovered after merging.
...
+
+1. Go to **Audience** > **Manage Audience** > **Edit rules**.
+2. In the **Profile to keep** section of the **Edit rules** panel, select the **Identifier** for the profiles that will be kept when merging duplicates. This can be the email address or phone number.
+3. In the **Resolving ties** section, select the criteria that will determine how to solve ties between profiles with matching criteria from **Profile to keep**. You can select the following:<br>
+- **Resolve ties using**: Created date, Updated date, Last session
+- **Prioritization**: Newest, Oldest
+
+![The "Edit rules" panel with sections to select options for "Profile to keep" and "Resolving ties".]({% image_buster /assets/img/audience_management/duplicate_users/edit_rules.png %})
+
+For example, you could keep the profile that has a phone number. If multiple users have the same phone number, you could resolve ties using the **Updated date** field and prioritze the newest user.
```
Comment (johndinh) [inline _docs/_user_guide/engagement_tools/segments/user_profiles/duplicate_users.md:105]: Updated date and newest would prioritize the most recently updated user.

Comment (rachel-feinberg) [inline _docs/_user_guide/engagement_tools/segments/user_profiles/duplicate_users.md:105]:
```suggestion
For example, you could keep the profile that has a phone number. If multiple users have the same phone number, you could resolve ties using the **Updated date** field and prioritize the most recently updated user.
```

**Review by johndinh — COMMENTED** [review]
Left one comment.  Once resolved, that should be good to go!

**Review by lydia-xie — APPROVED** [review]
lgtm!

**Pull request merged**